src/Models/BaseElement.php

```diff
@@ -1231,22 +1231,28 @@ public function getPageTitle()
         return null;
     }
 
-    /**
-     * @return boolean
-     */
-    public function First()
+    public function IsFirst(): bool
     {
         return ($this->Parent()->Elements()->first()->ID === $this->ID);
     }
 
-    /**
-     * @return boolean
-     */
-    public function Last()
+    public function First()
```

Comment:
```suggestion
    /**
     * @deprecated 5.4.0 Use IsFirst() instead.
     */
    public function First()
```
As per https://docs.silverstripe.org/en/6/developer_guides/deprecations/
Please also add this to `Last()`